packages/cli/test/command_builder.test.ts

```diff
@@ -27,20 +27,18 @@ describe('Command builder', () => {
       stringsList: string[]
     }
 
-    const action: jest.Mock<
-      Promise<CliExitCode>,
-      [DefActionInput<DummyCommandArgs>]
-    > = jest.fn().mockImplementation(async ({
-      input: { bool, string, choices, stringsList },
-    }): Promise<CliExitCode> => {
-      if (bool && string && choices && stringsList) {
-        if (string === 'error') {
-          throw new Error('error')
+    const action = mocks.mockFunction<CommandDefAction<DummyCommandArgs>>()
+      .mockImplementation(async ({
+        input: { bool, string, choices, stringsList },
+      }): Promise<CliExitCode> => {
```

Comment:
nit / fyi
now that you are using the typed `mockFunction` you don't have to specify the return type (it is inferred from the function type)
```suggestion
      }) => {
```